tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.md

```diff
@@ -1,113 +1,105 @@
----
```

Comment:
what happened to the YAML header for this file? this diff shows that it was all removed. I think that you should be able to re-knit this .md file to populate this YAML header again.

Reply:
So I was having an issue where the regular knit button wasn't updating the .R or the .md files at all (only the HTML). I tried running the knit function manually, which didn't work for some reason. I ended up using render() from the rmarkdown package, specifying the file type.

Basically looks like this:
HTML:
  `render(path, output_format = "html_document")`
md:
  `render(path, output_format = "md_document")`
R:
  `knit(path, tangle = TRUE)`            

No idea why the YAML header would be removed. Perhaps there's an argument for that? Will try to investigate a more detailed answer in the morning.